mirage/factories/vulnerability.js

```diff
@@ -3,20 +3,21 @@ import ENUMS from 'irene/enums';
 import Base from './base';
 
 export default Base.extend({
-  name: faker.company.companyName(),
-  description: faker.lorem.sentence(),
-  question: faker.lorem.sentence(),
-  successMessage: faker.lorem.sentence(),
-  moreInfo: faker.lorem.sentence(),
-  heading: faker.lorem.sentence(),
-  intro: faker.lorem.sentence(),
-  complaint: faker.lorem.sentence(),
-  nonComplaint: faker.lorem.sentence(),
-  relatedTo: faker.lorem.sentence(),
-  businessImplication: faker.lorem.sentence(),
+  uuid: () => faker.datatype.uuid(),
+  name: () => faker.company.companyName(),
+  description: () => faker.lorem.sentence(),
+  question: () => faker.lorem.sentence(),
+  'success-message': () => faker.lorem.sentence(),
```

Comment:
Just curious. Why this naming format?

Reply:
this is for json api response same format coming from backend